files/image_config/resolv-config/resolv-config.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+WD=/var/run/resolvconf/
+CONFIG_DIR=${WD}/interface/
+STATIC_CONFIG_FILE=mgmt.static
+DYNAMIC_CONFIG_FILE_TEMPLATE=*.dhclient
+
+start()
+{
+    redis-dump -d 4 -k "DNS_NAMESERVER*" -y > /tmp/dns.json
```

Comment:
>-d 4 [](http://example.com/codeflow?start=15&length=4)

If your intention is to check if the table is empty, no need to dump.

4 is magic number, you may consider `sonic-db-cli` with `CONFIG_DB`.

@judyjoseph to check multi-asic and T2.